docs/user-guide/connecting-applications.md

```diff
@@ -393,6 +393,165 @@ LoadBalancer Ingress:   a320587ffd19711e5a37606cf4a74574-1142138393.us-east-1.el
+  ports:
+  - port: 80
+    targetPort: 8080
+    protocol: TCP
+    name: http
+  selector:
+    app: echoheaders
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: echoheaders
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: echoheaders
+    spec:
+      containers:
+      - name: echoheaders
+        image: gcr.io/google_containers/echoserver:1.2
+        ports:
+        - containerPort: 8080
+```
+
+To make use of the Ingress resource, you need to first deploy an Ingress controller. On GCE/GKE there should already be an l7-controller deployed into the `kube-system` namespace:
+
+```shell
+$ kubectl get pods --namespace=kube-system -l name=glbc
+NAME                            READY     STATUS    RESTARTS   AGE
+l7-lb-controller-v0.6.0-chnan   2/2       Running   0          1d
+```
+
+Since Ingress is still a beta resource, make sure you review the [beta limitations](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/gce/BETA_LIMITATIONS.md) of the controller. In particular, you need to create a single firewall-rule on your cloudprovider, to allow health checks. On GKE this would be:
+
+```shell
+$ export TAG=$(basename `gcloud container clusters describe ${CLUSTER_NAME} --zone ${ZONE} | grep gke | awk '{print $2}'` | sed -e s/group/node/)
+$ export NODE_PORT=$(kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services echoheaders)
+$ gcloud compute firewall-rules create allow-130-211-0-0-22 \
+  --source-ranges 130.211.0.0/22 \
+  --target-tags $TAG \
+  --allow tcp:$NODE_PORT
+```
+
+In environments other than GCE/GKE, you need to [deploy a controller](https://github.com/kubernetes/contrib/tree/master/ingress/controllers).
+Once your controller is Running, you can create an Ingress:
+
+```yaml
+kind: Ingress
+metadata:
+  name: test
+spec:
+  # Catchall default backend for requests that don't match a host/url
+  backend:
+    serviceName: echoheaders
+    servicePort: 80
+  rules:
+  # Everything with a host header of example.com
+  - host: example.com
+    http:
+      paths:
+      - backend:
+          serviceName: nginxsvc
+          servicePort: 80
+  # Everything that's not example.com goes to default backend unless the path
+  # ends in /nginx
+  - http:
+      paths:
+      - path: /nginx
+        backend:
+          serviceName: nginxsvc
+          servicePort: 80
+```
+
+And wait till it acquires an IP:
+
+```yaml
+$ kubectl get ing
+NAME      RULE          BACKEND          ADDRESS        AGE
+test      -             echoheaders:80   130.211.5.76   2m
+          example.com
+                        nginxsvc:80
+
+          /nginx        nginxsvc:80
+```
+
+You can get more detailed information about the state of the Ingress through `kubectl describe`. This output is controller specific. The GCE controller for example, will annotate the output of `kubectl describe` with information about the backend health checks. Requests will fail if backends are not healthy.
```

Comment:
Go ahead and link to the docs for kubectl describe, which are at this URL

/docs/user-guide/kubectl/kubectl_describe/